CHANGES.md

```diff
@@ -1,36 +1,40 @@
 ## Master
 
-New language:
+New languages:
 
-- *isbl* built-in language DIRECTUM and Conterra by [Dmitriy Tarasov][].
-- *ReasonML* by [Gidi Meir Morris][]
 - *AngelScript* by [Melissa Geels][]
 - *GML* by [meseta][]
+- *isbl* built-in language DIRECTUM and Conterra by [Dmitriy Tarasov][].
 - *PostgreSQL* SQL dialect and PL/pgSQL language by [Egor Rogov][].
+- *ReasonML* by [Gidi Meir Morris][]
 
-New style:
-
-- *isbl editor light* by [Dmitriy Tarasov][]
+New styles:
+- *An Old Hope* by [Gustavo Costa][]
+- *Atom One Dark Reasonable* by [Gidi Meir Morris][]
 - *isbl editor dark* by [Dmitriy Tarasov][]
+- *isbl editor light* by [Dmitriy Tarasov][]
 - *Lightfair* by [Tristian Kelly][]
 - [*Nord*][nord-highlightjs] by [Arctic Ice Studio][]
-- *Atom One Dark Reasonable* by [Gidi Meir Morris][]
+- *Shades of Purple* by [Ahmad Awais][]
```

Comment:
Can you change this like this

```diff
- *Shades of Purple* by [Ahmad Awais][]
+ *[🦄 Shades of Purple](https://github.com/ahmadawais/Shades-of-Purple-HighlightJS)* by [Ahmad Awais][https://github.com/AhmadAwais]
```

Reply:
Looks like it's merged already so I'll send a PR.

Reply:
Thanks